internal/postprocessor/main.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/fs"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// Vet runs linters on all .go files recursively from the given directory.
+func Vet(dir string) error {
+	log.Println("vetting generated code")
+	c := exec.Command("gofmt", "-s", "-d", "-w", "-l", ".")
+	c.Dir = dir
+	return c.Run()
+}
+
+// ModTidy tidies go.mod file in the specified directory.
+func ModTidy(dir string) error {
+	c := exec.Command("go", "mod", "tidy")
+	c.Dir = dir
+	return c.Run()
+}
+
+// ModTidyAll tidies all mod files from the specified root directory.
+func ModTidyAll(dir string) error {
+	log.Printf("[%s] finding all modules", dir)
+	var modDirs []string
+	err := filepath.WalkDir(dir, func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if d.Name() == "go.mod" {
+			modDirs = append(modDirs, filepath.Dir(path))
+		}
+		return nil
+	})
+	if err != nil {
+		return err
+	}
+	for _, modDir := range modDirs {
+		if err := ModTidy(modDir); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func copyFiles(srcPath, dstPath string) error {
+	src, err := os.Open(srcPath)
+	if err != nil {
+		return err
+	}
+
+	dst, err := os.Create(dstPath)
+	if err != nil {
+		return err
+	}
+	defer dst.Close()
+
+	nBytes, err := io.Copy(dst, src)
```

Comment:
You are right that we don't need to look at nBytes here. Instead of ignoring it in the next line just use the `_` here.